### PR TITLE
check install automake and auto conf

### PR DESCRIPTION
**Pull request by saturn99**
https://github.com/tinyproxy/tinyproxy/issues/204

### CODE REVIEW

File: autogen.sh [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+checkInstalled () {
+	which "$1" || { echo "$1 not installed"; exit 0; }
```
Comment (rofl0r) [inline autogen.sh]: `which` is not POSIX.
IIRC `type` is, but what i'd do here instead is simply trying to run the command with --help
i.e.
`$1 --help >/dev/null 2>&1 || ...`

Comment (rofl0r) [reply autogen.sh]: also the exit returnvalue needs to be 1

Comment (saturn99) [reply autogen.sh]: > also the exit returnvalue needs to be 1

:+1: :D

**Comment by rofl0r** [comment]
?

**Pull request closed without merging**